ci/appveyor/linux_prepare.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+sudo apt update
+# sudo apt-get upgrade -y
+sudo apt install -y git curl libssl1.0.0 build-essential libsodium23 libsodium-dev autoconf ncurses-dev libssl-dev
```

Comment:
Would it be possible to make this as close as possible to https://github.com/aeternity/docker-builder/blob/23756220edbca600f3f7af53b0ee7c8f7127cad1/Dockerfile#L4-L7 and link that here? This way we would avoid to think maintaining two different sets of scripts.

Reply:
Its somewhat different though since Erlang is not compiled from source in this setup.